backend/notification_v2/provider/webhook/webhook.py

```
@@ -51,6 +52,13 @@ def validate(self):
         """
         if not self.notification.url:
             raise ValueError("Webhook URL is required.")
+
+        # Validate webhook URL for security
+        is_valid, error_message = URLValidator.validate_url(self.notification.url)
+        logger.info(f"Notification url {self.notification_url}")
+        if not is_valid:
+            raise ValueError(f"Webhook URL validation failed: {error_message}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)